dietpi/dietpi-config

```diff
@@ -1112,15 +1112,15 @@
 
 			#Get Current Values
 			local current_resolution=''
-			if (( $(grep -m1 'setenv videoconfig "' /DietPi/boot.ini | grep -ci -m1 1920x1080) == 1 )); then
+			if grep -m1 'setenv videoconfig "' /DietPi/boot.ini | grep -qi 1920x1080; then
```

Comment:
I'll try and clean this up, old code, seems excessive?

Reply:
@Fourdee 
Jep, single grep, combining both patterns with wildcard will work. As said, I became just a bid tired and lazy and didn't want to accidentally break something, where I am not 100% sure 😉.